dallinger/recruiters.py

```diff
@@ -145,6 +154,26 @@ def recruit_participants(self, n=1):
             duration_hours=self.config.get('duration')
         )
 
+    def notify_recruited(self, participant, experiment):
+        """Assign a Qualification to the Participant for the experiment ID,
+        and for the configured group_name, if it's been set.
+        """
+        score = '1'  # always '1'; doesn't matter
```

Comment:
@jessesnyder Let's add it as a new story unless you'd rate it as a 1- or 2-point story or if you think that the context-switching cost is high enough that it'd be best to just do it now.